### ISSUE TITLE
Crashes on test with xfail

### ISSUE THREAD

**Issue by darrinwillis** [python-tap/pytest-tap#22]
The handling of skipped tests makes the assumption that 'longrepr' is a tuple, which is not the case for `xfail`ed tests.

Error:
```
INTERNALERROR>   File "...../pytest_tap/plugin.py", line 62, in pytest_runtest_logreport
INTERNALERROR>     reason = report.longrepr[2].split(':', 1)[1].strip()
INTERNALERROR> AttributeError: ReprExceptionInfo instance has no attribute '__getitem__'

```
Offending code:
```
def pytest_runtest_logreport(report):
...
    elif report.outcome == 'skipped':
        reason = report.longrepr[2].split(':', 1)[1].strip()
```
Example of a test that crashes:
```
import pytest
@pytest.mark.xfail()
def test_fail():
    assert 1 == 2
```

**Comment by mblayman** [comment]
Thanks for the excellent bug report. I'm not deep in pytest (I'm more of a nosetests user) so this is very useful. I'll see about getting a fix out there in the next few days.

**Comment by mblayman** [comment]
This fix was just released with the 2.0 version that I put out. Thanks again for the bug report!

**Comment by darrinwillis** [comment]
Thanks a ton for the quick fix! I can confirm that the above problem is now fixed.
I have a much more slight issue with the new behavior, but I'll open a new issue about it.
